homeassistant/components/trello/sensor.py

```diff
@@ -0,0 +1,82 @@
+"""Platform for sensor integration."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import SensorEntity, SensorStateClass
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, Board, List
+from .coordinator import TrelloDataUpdateCoordinator
+
+
+class TrelloSensor(CoordinatorEntity[TrelloDataUpdateCoordinator], SensorEntity):
+    """Representation of a TrelloSensor."""
+
+    _attr_native_unit_of_measurement = "Cards"
+    _attr_state_class = SensorStateClass.MEASUREMENT
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        board: Board,
+        list_: List,
+        coordinator: TrelloDataUpdateCoordinator,
+    ) -> None:
+        """Initialize sensor."""
+        super().__init__(coordinator)
+        self.board = board
+        self.list_id = list_.id
+        self._attr_unique_id = f"list_{list_.id}".lower()
+        self._attr_name = list_.name
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, board.id)},
+            name=board.name,
+            entry_type=DeviceEntryType.SERVICE,
+            manufacturer="Trello",
+            model="Board",
+        )
+
+    @property
+    def available(self) -> bool:
+        """Determine if sensor is available."""
+        board = self.coordinator.data[self.board.id]
```

Comment:
```suggestion
	if not super.available():
	    return False
        board = self.coordinator.data[self.board.id]
```

We need to call it `super.available()` otherwise it could happen that we get a `KeyError` on accessing `self.coordinator.data`